app/(auth)/actions.ts

```diff
@@ -48,7 +48,7 @@ export async function signup(formData: FormData) {
   }
 
   revalidatePath("/", "layout");
-  redirect("/");
+  redirect("/signin");
```

Comment:
Does this redirect to sign in after user signs up?

Reply:
Yes

Reply:
Hmm could we have the user signed in automatically after they signed up? If complicated then no need.

Reply:
Yes, but that will require disabling email confirmation.